index.js

```diff
@@ -47,21 +62,41 @@ const getRealType = (value) => {
     //     getRealType(NaN)        // 'NaN'
     // Use typeof, instanceof and some magic. It's enough to have
     // 12-13 unique types but you can find out in JS even more :)
+    if (Number.isNaN(value)) return 'NaN';
+    if (typeof value === 'number' && !isFinite(value)) return 'Infinity';
+    else
+        return {}.toString
```

Comment:
Можно так для краткости, но правильнее было бы напрямую использовать этот метод:
```suggestion
        return Object.prototype.toString
```
И объектов лишних не создаём.

Reply:
fixed